.travis.yml

```diff
@@ -45,6 +46,7 @@ install:
   - gem install bundler
   - bundle install
   - git clone https://github.com/DataDog/dd-agent.git ~/dd-agent
+  - cp ~/dd-agent/datadog.conf.example ~/dd-agent/datadog.conf
```

Comment:
We don't need this, all we need to do is mock `get_logging_config()` to return a valid logging configuration. That's the function in JMXFetch that is causing the issue. It's not a big deal, so feel free to ignore. Just saying because the less clutter the better.